app/views/calendar/_create_event.erb

```diff
@@ -0,0 +1,50 @@
+<div class="modal fade"
+     id="create-event-modal"
+     tabindex="-1"
+     role="dialog"
+     aria-labelledby="create-event-modal-label">
+  <div class="modal-dialog" role="document">
+    <%= form_for :event, url: events_path, remote: true, html: {id: 'create-event-form'} do |e| %>
```

Comment:
you might want to give it a try with restful forms:
`<%= form_for Event.new, remote: true, html: {id: 'create-event-form'} do |e| %>`